lib/jquery/jquery.inputautoexpand.js

```diff
@@ -320,7 +315,8 @@ $.extend( $.AutoExpandInput.prototype, {
 	_getComfortZone: function() {
 		return ( this._options.comfortZone )
 			? this._options.comfortZone
-			: Math.ceil( this._getWidthFor( ' ' ) * 2 );
+			// IE does not recognize a single space character, using some narrow character instead:
+			: Math.ceil( this._getWidthFor( 'I' ) * 2 );
```

Comment:
The width of an "I" is very diverse across fonts. I find characters like "e" (as in `em`) or "W" more reliable.